src/Models/News.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Models;
+
+use Blumilk\Meetup\Core\Formats;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+class News extends Model
```

Comment:
I think a better model name is a Newsletter

Reply:
No, it's not, becouse it's News Model from #14  not Newsletter